src/frontend/src/env/networks.icrc.env.ts

```diff
@@ -462,9 +458,18 @@ export const ICRC_TOKENS: IcInterface[] = [
 	...(nonNullish(CKUNI_IC_DATA) ? [CKUNI_IC_DATA] : []),
 	...(nonNullish(CKEURC_IC_DATA) ? [CKEURC_IC_DATA] : []),
 	...(nonNullish(CKXAUT_IC_DATA) ? [CKXAUT_IC_DATA] : [])
+]
+
+const ADDITIONAL_ICRC_TOKENS: IcInterface[] = [...(nonNullish(BURN_IC_DATA) ? [BURN_IC_DATA] : [])];
+
+export const ICRC_TOKENS: IcInterface[] = [
+	...PUBLIC_ICRC_TOKENS,
```

Comment:
Shall we also rename this to PUBLIC_CK_ICRC_TOKENS as it only holds CKBTC, CKETH and CKUSDC at the moment?